#nebula_velodyne_hw_interface:
#nebula_velodyne_hw_monitor:
/**:
  ros__parameters:
    sensor_model: "VLP32" # See readme for supported models
    sensor_ip: "192.168.1.201" # Lidar Sensor IP
    host_ip: "255.255.255.255" # Broadcast IP from Sensor
    frame_id: "velodyne"
    data_port: 2368 # LiDAR Data Port
    gnss_port: 2369 # LiDAR GNSS Port
    return_mode: "Dual" # See readme for supported return modes
    scan_phase: 0.0 # Angle where scans begin (degrees, [0.,360.]
    packet_mtu_size: 1500 # Packet MTU size
    rotation_speed: 600 # Motor RPM, the sensor's internal spin rate.
    cloud_min_angle: 0 # Field of View, start degrees.
    cloud_max_angle: 359 # Field of View, end degrees.
    diag_span: 1000 # milliseconds
    advanced_diagnostics: False # original diag
    min_range: 0.3
    max_range: 300.0
    view_width: 360.0
    calibration_file: "./install/nebula_lidar_driver/share/nebula_lidar_driver/calibration/velodyne/VLP32.yaml"

    online: True
  